librz/core/cbin.c

```diff
@@ -1854,6 +1854,9 @@ RZ_API void rz_core_bin_print_source_line_info(RzCore *core, const RzBinSourceLi
 }
 
 static const char *bin_reloc_type_name(RzBinReloc *reloc) {
+	if (reloc->print_name != NULL) { // print_name is set to NULL by default. If it's not null, that means we can use the new naming
```

Comment:
```suggestion
	if (reloc->print_name) {
```

This is a very common pattern in C code. No need to document it :)